File: src/services/EventService.js
import axios from 'axios';

const REST_API_BASE_URL = 'http://localhost:8080/api/events';

export const listEvents = () => axios.get(`${REST_API_BASE_URL}/all-events`);
export const createEvent = (event) => axios.post(`${REST_API_BASE_URL}`, event);

File: eventmanagement/eventmanagement/src/main/java/event/management/eventman/service/EventService.java
package event.management.eventman.service;

import event.management.eventman.dto.EventDto;
import event.management.eventman.entity.Event;

import java.util.List;

public interface EventService {

    EventDto createEvent(EventDto eventDto, String userId);

    List<EventDto> getEventsByUserId(String userId);
    List<EventDto> getAllEventsExceptUserId(String userId);
    void deleteEventById(String eventId);
    List<EventDto> getAllEvents(); // New method to get all events




}
